docs-partials/builder/common/AccessConfig-required.mdx

```diff
@@ -1,7 +1,7 @@
 <!-- Code generated from the comments of the AccessConfig struct in builder/common/access_config.go; DO NOT EDIT MANUALLY -->
```

Comment:
Hello, thanks for openning, this file is autogenerated from `builder/common/access_config.go`, you might want to update the file there and then run `make generate`.